sw/airborne/modules/guidance/gvf/gvf.h

```diff
@@ -32,6 +32,13 @@
 
 #include "std.h"
 
+/** \typedef gvf_conf
```

Comment:
I think this would work, although I'm not sure about the `\param` keyword... also we usually use the `@` javadoc notation, see http://docs.paparazziuav.org/latest/styledoxygen.html

You could write this in an easier way though:
```
/**
 * Parameters for the GVF
 */
typedef struct {
   float ke; ///< Gain defining how agressive is the vector field
   float kn; ///< Gain for making converge the vehile to the vector field
   float error; ///< Error signal (unit ?)
   int8_t s; ///< Right or left direction (and how is left/right encoded?)
} gvf_con;
```

Also please add the units of the variables in the description.